src/c++/perf_analyzer/genai-pa/genai_pa/main.py

```diff
@@ -88,7 +88,7 @@ def run(argv=None):
         args, extra_args = parser.parse_args(argv)
         generate_inputs(args)
         args.func(args, extra_args)
-        metrics = calculate_metrics(args.profile_export_file)
+        metrics = calculate_metrics(args.profile_export_file, args.service_kind)
```

Comment:
are you going to need args.endpoint as well? (it doesn't exist yet. It is in a side branch that Elias has. It would tell you chat/completions vs legacy completions)

Reply:
openai response has information about it as well so I'm using that: https://github.com/triton-inference-server/client/pull/493/files#diff-adea1b0fba86455c43bc4414f2eef716b93ddd24e3240cb2b9190091a2806dbcR294